regulations/static/regulations/css/less/module/drawer-content.less

```diff
@@ -90,8 +90,8 @@ This contains all of the styles specific to the TOC
 
   a:link,
   a:visited {
-    color: @black;
-    .border-bottom-light (@width: 1px);
+    color: @navigation_gray;
+    .border-bottom-light (@width: 0px);
```

Comment:
It should be applied to everything in the ToC for both the the Preamble and CFR tabs so we could either consider those to be the exception or ATF use case to be the exception.